### ISSUE TITLE
Blokada contacting amplitude.com

### ISSUE THREAD

**Issue by kreisklasse** [blokadaorg/blokada#322]
### Expected behaviour

Blokada should not contact any tracking urls

### Actual behaviour

Blokada contacted _journal.blokada.org_ which translates to _api.amplitude.com_

see pihole logfile
>Feb  2 00:07:17 dnsmasq[15299]: query[A] journal.blokada.org from 192.168.1.1
>Feb  2 00:07:17 dnsmasq[15299]: forwarded journal.blokada.org to 104.207.131.11
Feb  2 00:07:17 dnsmasq[15299]: reply journal.blokada.org is <CNAME>
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 34.212.78.203
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 34.218.122.196
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 52.11.211.245
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 35.162.160.16
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 34.214.40.205
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 52.25.239.67
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 34.210.236.76
Feb  2 00:07:17 dnsmasq[15299]: reply api.amplitude.com is 35.166.126.186

Amplitude.com
>Amplitude is the comprehensive product analytics software for web and mobile.

Contacting this 3rd-party is not mentioned in the [Terms&Conditions](https://blokada.org/terms/)

### Configuration
1. Blokada version
3.6 (101900)

### Questions
1. How long have you been using Blokada? 
> 2 month

3.  How often does this problem happen to you?
minimum every 24 hours

**Comment by rpeter85** [comment]
hi,
By any chance, did your Blokada crash recently? The crash reports are collected to see what lead to that, what issues can be found in the log to change the application's behaviour based on them.

**Comment by kreisklasse** [comment]
No, Blokada ran since several days. I only switched it off days before to use a other vpn-software.

Would be more privacy friendly when Blokada would have an opt-in for sending logs. Or asking about it, including to show the logs to send in advance.

**Comment by rpeter85** [comment]
Good idea, I like it. I don't know how much of the users would check the log before sending that, but as an option it would be great.

**Comment by rpeter85** [comment]
hi,
do you mind to close this thread and open a feature request for the op-out function amd review of the logs before sending them?
